cgp/cartesian_graph.py

```diff
@@ -230,24 +230,35 @@ def _fill_parameter_values(self, func_str: str) -> str:
                 )
         return func_str
 
-    def to_func(self) -> Callable[[List[float]], List[float]]:
-        """Compile the function(s) represented by the graph.
+    def to_func(self) -> Callable[..., List[float]]:
+        """Create a Python callable implementing the function described by
+        this graph.
 
-        Generates a definition of the function in Python code and
-        executes the function definition to create a Callable.
+        The function expects as many arguments as the number of inputs
```

Comment:
Same as below indicated by @HenrikMettler : To be clear and consistent with the wording, I would use "The returned Callable" here and in the rest of the docstring.